client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -45,9 +47,10 @@ const MetadataEditor = createClass({
 	},
 
 	handleFieldChange : function(name, e){
-		this.props.onChange(_.merge({}, this.props.metadata, {
+		this.props.onChange({
+			...this.props.metadata,
```

Comment:
I do like this better. There's no functional difference you've noticed doing it with the spread operator vs. `merge` with an empty object?